src/main/java/athleticli/parser/Parser.java

```diff
@@ -120,6 +124,8 @@ public static Command parseCommand(String rawUserInput) throws AthletiException
             return new FindActivityCommand(parseDate(commandArgs));
         case CommandName.COMMAND_ACTIVITY_GOAL_SET:
             return new SetActivityGoalCommand(ActivityParser.parseActivityGoal(commandArgs));
+        case CommandName.COMMAND_ACTIVITY_GOAL_DELETE:
+            return new DeleteActivityGoalCommand(ActivityParser.parseDeleteActivityGoal(commandArgs));
```

Comment:
Good feature, but lets not introduce any more new commands and features after the feature freeze, as that would create more testing required.

Reply:
Yes good idea and this feature was proposed and on the bucket list last week. I only got the chance to implement it now.